innovation_sweet_spots/pipeline/pilot/investment_predictions/link_cb_gtr/link_cb_gtr.py

```diff
@@ -0,0 +1,130 @@
+from innovation_sweet_spots import PROJECT_DIR
```

Comment:
Perhaps useful to add some usage instructions at the very top? Something to the effect of:

```suggestion
"""
Script to link organisations in the GtR and Crunchbase datasets by using fuzzy matching

Run the following command in the terminal to see the options for matching the datasets:
python innovation_sweet_spots/pipeline/pilot/investment_predictions/link_cb_gtr/link_cb_gtr.py --help
"""
from innovation_sweet_spots import PROJECT_DIR
```

Reply:
Added, thanks.